requirements.txt

```diff
@@ -1,5 +1,4 @@
-torch>=1.6
+torch>=1.7.*
```

Comment:
`>=1.7.*` on purpose rather than `>=1.7` for the same reason as https://github.com/PyTorchLightning/pytorch-lightning/pull/10367#issuecomment-961916435.